src/js/dropdown-menu/dropdown-menu-view.js

```diff
@@ -102,7 +114,22 @@ define([
             },
 
             clickCloseDropdown: function(event, context) {
-                var $el = $(event.target);
+                var $el = $(event.target),
```

Comment:
I thought from the way the event was registered, that it was for items on the page with the class ``dropdown``. Can you double check that ``currentTarget`` isn't what you want, because it would simplify this logic.